code/modules/paperwork/sticky_note.dm

```diff
@@ -9,13 +9,13 @@
 
 /obj/item/paper_bin/pad/examine(mob/user)
 	. = ..()
-	. += span_notice("Write notes on it using a <b>pen</b>.")
+	. += span_notice("Write notes on top using a <b>pen</b>.")
 
 /obj/item/paper_bin/pad/Initialize()
 	. = ..()
 	var/random_color = pick(COLOR_YELLOW, COLOR_LIME, COLOR_CYAN, COLOR_ORANGE, COLOR_PINK)
 	for(var/obj/item/paper/sticky/note in contents)
-		note.color = random_color
+		note.color = color ? color : random_color
```

Comment:
Can this be moved to when the sticky note is removed from the box? This is pretty expensive,